src/kubernetes.cc

```diff
@@ -993,8 +1052,9 @@ void KubernetesReader::WatchPods(MetadataUpdater::UpdateCallback callback)
                           std::placeholders::_2, std::placeholders::_3));
   } catch (const json::Exception& e) {
     LOG(ERROR) << e.what();
+    LOG(ERROR) << "No more pod metadata will be collected";
   } catch (const KubernetesReader::QueryException& e) {
-    // Already logged.
+    LOG(ERROR) << "No more pod metadata will be collected";
```

Comment:
Distinguish the log of these two errors for debugging purpose? Same below.

Reply:
Ah, I see. Then we should be fine.

Reply:
We're not logging the `e.what()` here as we do above, is that because it's logged elsewhere? Can we juggle things around for consistency?

Reply:
Yes, it's because it's logged elsewhere. The code is fairly consistent about logging any time a `QueryException` is thrown, so we don't need to log when catching it.
I've just added one missing log statement above.